src/docs/ad/ad-service-discovery.rst

```diff
@@ -0,0 +1,84 @@
+Service Discovery
+#################
+
+Active Directory Terms
+======================
+
+`Active Directory <https://en.wikipedia.org/wiki/Active_Directory>`_ is a directory service providing Windows and Linux clients with information about centrally stored user and group information. Below are some Active Directory terms that are important to understand for the purposes of this document.
+
+Domain Controller
+    The Active Directory Domain Controller (DC for short) is an important source of user information and authentication in many computing environments. The DC stores user and group information in a LDAP directory server (database) and provides this information as a `service <https://docs.microsoft.com/en-us/windows-server/identity/ad-ds/get-started/virtual-dc/active-directory-domain-services-overview>`_. This allows administrators to configure clients to retrieve user information from the AD DC. Similarly, the DC contains Kerberos data used to authenticate users password and access to other network services.
```

Comment:
a LDAP -> an LDAP?

Reply:
I don't believe so - https://owl.purdue.edu/owl/general_writing/grammar/articles_a_versus_an.html

Reply:
Isn't it driven by the sound instead of written form? I.e. "an eldap" instead of "a eldap", like "an honorable piece" or "a union" from the listed exceptions?

Reply:
I don't want to teach you your language, this is what I have been taught so I'm curios.

Reply:
You are right, "If the first letter makes a vowel-type sound" I was just looking at LDAP but not thinking about this. Fixed to 'an LDAP'